### PR TITLE
[evm] implement equality for vector of primitives

### PR DESCRIPTION
**Pull request by emmazzz**
## Motivation

This PR adds to move-to-yul compiler equality comparison for vectors of primitives, such as `vector<u8>`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 # The Move Language
 
-Move is a programming language for writing safe smart contracts originally developed at Facebook to power the Diem blockchain. Move is designed to be a platform-agnostic language to enable common libraries, tooling, and developer communities across diverse blockchains with vastly different data and execution models. Move's ambition is to become the "JavaScript of web3" in terms of ubiquity--when developers want to quickly write safe code involving assets, it should be written in Move. 
+Move is a programming language for writing safe smart contracts originally developed at Facebook to power the Diem blockchain. Move is designed to be a platform-agnostic language to enable common libraries, tooling, and developer communities across diverse blockchains with vastly different data and execution models. Move's ambition is to become the "JavaScript of web3" in terms of ubiquity--when developers want to quickly write safe code involving assets, it should be written in Move.
```
Comment (emmazzz) [inline README.md:8]: Had to remove trailing whitespace here for lint to pass.

**Comment by wrwg** [comment]
Please rebase and push again, it should pass the checks then.

**Review by wrwg — APPROVED** [review]

**Review by junkil-park — APPROVED** [review]
Thanks!

**Comment by junkil-park** [comment]
Wow. It's quick. Apparently, we are not using the `/land` command any longer?

**Comment by junkil-park** [comment]
this PR resolves #15 and #16 .

**Pull request merged**